assets/js/wcmp-admin.js

```diff
@@ -867,3 +848,36 @@ jQuery(function($) {
   }
 });
 
+/**
+ * Object.assign() polyfill.
+ */
+if (typeof Object.assign !== 'function') {
+  // Must be writable: true, enumerable: false, configurable: true
```

Comment:
Als de code geminified is, zal `//` voor problemen zorgen 
```suggestion
  /* Must be writable: true, enumerable: false, configurable: true */
```